app/Http/Controllers/ArticleController.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Article;
+use Illuminate\Http\Request;
+
+class ArticleController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        // ④DBから記事情報を取得して変数に代入。
+        // $articles = Article::all();
+
+        // 更新した順に並び替え
+        $articles = Article::orderby('created_at', 'desc')->get();
+
+        //③記事一覧画面を表示。articleというリンク先（名前？）についてはlayouts/dashboardというものをviewさせるという意味になる！compactには上記の変数を入れており、DBからのデータを反映させている。
+        return view('layouts/dashboard', compact('articles'));
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
```

Comment:
```suggestion
     * @return \Illuminate\Http\Response
```
ここも上のコメントと同じで、型が違うね。上と一緒のになおしといて

Reply:
<img width="784" alt="スクリーンショット 2022-12-05 23 26 56" src="https://user-images.githubusercontent.com/91363928/205661352-c45e4650-dd9d-4825-8c81-b2f940edd31f.png">
上記の指摘通りに修正したが、そのままでよいか。

Reply:
@return \Illuminate\Http\RedirectResponseに修正済。